src/mumble/Settings.h

```diff
@@ -379,34 +380,34 @@ struct Settings {
 	/// Name of the style to use from theme. @see Themes
 	QString themeStyleName = QStringLiteral("Lite");
 
-	QByteArray qbaMainWindowGeometry     = {};
-	QByteArray qbaMainWindowState        = {};
-	QByteArray qbaMinimalViewGeometry    = {};
-	QByteArray qbaMinimalViewState       = {};
-	QByteArray qbaHeaderState            = {};
-	QByteArray qbaConfigGeometry         = {};
-	WindowLayout wlWindowLayout          = LayoutClassic;
-	ChannelExpand ceExpand               = ChannelsWithUsers;
-	ChannelDrag ceChannelDrag            = Ask;
-	ChannelDrag ceUserDrag               = Move;
-	bool bMinimalView                    = false;
-	bool bHideFrame                      = false;
-	AlwaysOnTopBehaviour aotbAlwaysOnTop = OnTopNever;
-	bool bAskOnQuit                      = true;
-	bool bEnableDeveloperMenu            = false;
-	bool bLockLayout                     = false;
-	bool bHideInTray                     = false;
-	bool bStateInTray                    = true;
-	bool bUsage                          = true;
-	bool bShowUserCount                  = false;
-	bool bShowVolumeAdjustments          = true;
-	bool bShowNicknamesOnly              = false;
-	bool bChatBarUseSelection            = false;
-	bool bFilterHidesEmptyChannels       = true;
-	bool bFilterActive                   = false;
-	QByteArray qbaConnectDialogHeader    = {};
-	QByteArray qbaConnectDialogGeometry  = {};
-	bool bShowContextMenuInMenuBar       = false;
+	QByteArray qbaMainWindowGeometry                          = {};
+	QByteArray qbaMainWindowState                             = {};
+	QByteArray qbaMinimalViewGeometry                         = {};
+	QByteArray qbaMinimalViewState                            = {};
+	QByteArray qbaHeaderState                                 = {};
+	QByteArray qbaConfigGeometry                              = {};
+	WindowLayout wlWindowLayout                               = LayoutClassic;
+	ChannelExpand ceExpand                                    = ChannelsWithUsers;
+	ChannelDrag ceChannelDrag                                 = Ask;
+	ChannelDrag ceUserDrag                                    = Move;
+	bool bMinimalView                                         = false;
+	bool bHideFrame                                           = false;
+	AlwaysOnTopBehaviour aotbAlwaysOnTop                      = OnTopNever;
+	WhenConnectedQuitBehaviour wcqbWhenConnectedQuitBehaviour = WhenConnectedQuitBehaviourAsk;
```

Comment:
Don't use Hungarian style naming conventions. I know lots of legacy code does this, but it's just horrible. 

Instead just call it e.g. `whenConnectedQuitBehavior`
Though, this is still a rather bulky name. I have to wonder whether we could come up with something more compact :thinking:

Reply:
I didn't want to use my own naming convention and used "AlwaysOnTopBehaviour" as a 'template'. So, how about ConnQuitBehaviour or ConnectedQuitBehaviour?

Reply:
> ConnectedQuitBehaviour

Yeah, that sounds better :+1: